addons-cluster/clickhouse/Chart.yaml

```diff
@@ -6,15 +6,14 @@ type: application
 
 version: 1.0.0-alpha.0
 
-appVersion: 22.9.4
+appVersion: 24.3.10
 
 dependencies:
   - name: kblib
     version: 0.1.1
-    repository: file://../kblib
+    repository: file://../kblib-v2
```

Comment:
the main branch have renamed kblib-v2 to kblb and removed the legacy kblib， so here should be repository: file://../kblib